modules/atf_logger.lua

```diff
@@ -1,10 +1,37 @@
 local json = require("json")
 local Logger = {}
 Logger.mobile_log_format = "%s(%s) [version: %s, frameType: %s, encryption: %s, serviceType: %s, frameInfo: %s, messageId: %s] : %s \n"
-Logger.hmi_log_format = "%s(%s) : %s \n"
+Logger.hmi_log_format = "%s[%s] : %s \n"
+
+
+function capture_cmd_output(cmd, is_raw)
+  local f = assert(io.popen(cmd, 'r'))
```

Comment:
@LevchenkoS,
Using *running system command* for *each* logging looks redundant.
Please, find more appropriate way to do that.
Assume writing separate `.so` or (better) updating ATF C++ Core.